### ISSUE TITLE
Build for web not working  –  The integer literal 9223372036854775807 can't be represented exactly in JavaScript.

### ISSUE THREAD

**Issue by jans-y** [pichillilorenzo/flutter_inappwebview#429]
## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |  v1.19.0-5.0.pre-dev  |
| Plugin version       |   3.4.0 - 4.0.0  |

## Description

**Expected behavior:** 
Build for web without issues

**Current behavior:**

```
Target dart2js failed: Exception:                                       
../../v1.19.0-5.0.pre-dev/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0/lib/src/X509Certificate/asn1_decoder.dart
:163:25:
Error: The integer literal 9223372036854775807 can't be represented exactly in JavaScript.
    int int64MaxValue = 9223372036854775807;
                        ^^^^^^^^^^^^^^^^^^^
Error: Compilation failed.
```

## Steps to reproduce

Hi,

I know this project doesn't support web but we have an app that runs on the mobile and web.

For web we are using dev channel since it has a lot of fixes over stable channel.

Currently It's not possible to build for the web.

Thank you.

**Comment by barriehadfield** [comment]
I am seeing exactly the same thing. Just started today.

**Comment by pichillilorenzo** [comment]
Fix will be available in the next release! Probably today or tomorrow

**Comment by pichillilorenzo** [comment]
Fixed with new release `4.0.0+4`
